projects/angular/src/layout/vertical-nav/_vertical-nav.clarity.scss

```diff
@@ -77,6 +77,7 @@
     //Display
     display: flex;
     flex-direction: column;
+    gap: 0;
```

Comment:
As discussed offline, these `gap: 0` probably wouldn't be needed when we revert the change for the anchor.

Reply:
corrected in commit bf22d26